src/coco/config/fileChange/configRenameUtil.ts

```diff
@@ -0,0 +1,69 @@
+import * as vscode from "vscode";
+import { wrapWithLogs } from "../../../log/util/logUtil";
+import FileChanges from "../../../common/vscode/fileChange/FileChanges";
+import { isInWorkspace } from "../../../common/vscode/workspace/workspaceUtil";
+import CustomConfigs from "../util/CustomConfigs";
+import MessageBuilder from "../../../common/vscode/message/MessageBuilder";
+import MessageType from "../../../common/vscode/message/MessageType";
+import localization from "../../../localization";
+import ConfigPaths from "../util/ConfigPaths";
+
+function updatePathOnCustomConfigRename(event: vscode.FileRenameEvent) {
+    event.files.forEach(
+        fileChanges => wrapWithLogs(
+            () => updateCustomConfigPaths(fileChanges),
+            "Updating path on custom config rename",
```

Comment:
I couldn't get the action in all perspectives but how about "Updating the custom config file's path."
If this makes sense we can set the copy below like "Removing custom config file's path."

Reply:
These logs are not visible to users. This is written on log file in case of an error and for our reference only. That's why it's short and informative (but not easy for users to understand). I can still change it the way you suggest, but I should add more information and it may become "Updating the custom config file's path because it's renamed"